src/Protocol/Diaspora.php

```diff
@@ -292,7 +292,7 @@ public static function decodeRaw($importer, $raw)
 			$ciphertext = base64_decode($data->encrypted_magic_envelope);
 
 			$outer_key_bundle = '';
-			@openssl_private_decrypt($encrypted_aes_key_bundle, $outer_key_bundle, $importer['prvkey']);
+			@Crypto::opensslPrivateDecrypt($encrypted_aes_key_bundle, $outer_key_bundle, $importer['prvkey']);
```

Comment:
There is no need applying this to the Diaspora part, since Diaspora doesn't work that way.

Reply:
I like the idea of using the same wrapper everywhere.

Reply:
This adds additional complexity, speed decrease and possible side effects.